README.md

```diff
@@ -14,78 +16,78 @@ nRF Connect supports the following devices:
 * PCA10059 nRF52840 Dongle
 * PCA10090 nRF9160 Development Kit
 
-Note that while the nRF Connect framework has support for all these devices, the device support in nRF Connect apps may vary.
-
-# Installation
-
-To install nRF Connect you can download binaries from the [nRF Connect product page](https://www.nordicsemi.com/eng/Products/Bluetooth-low-energy/nRF-Connect-for-Desktop#Downloads) on Nordic Semiconductor web pages.
+Note that while the the framework has support for all these devices, the device support in individual apps may vary.
 
-nRF Connect currently supports the following operating systems:
+You can run nRF Connect for Desktop on:
 
 * Windows
 * Ubuntu Linux 64-bit
 * macOS
 
-# Usage documentation
+# Using nRF Connect for Desktop
+
+## Prerequisites
+
+### macOS
 
-A [Getting started guide](https://www.nordicsemi.com/eng/Products/Bluetooth-low-energy/nRF-Connect-for-Desktop#Documentation) is available from the nRF Connect product pages.
+J-Link driver needs to be separately installed on macOS. Download and install it from [SEGGER](https://www.segger.com/downloads/jlink) under the section *J-Link Software and Documentation Pack*. Without it when running the apps you would otherwise get error messages when running the apps like `CouldNotOpenDLL` or `JLINKARM_DLL_NOT_FOUND`.
+
+### Linux
+
+J-Link driver needs to be separately installed on Linux. Download and install it from [SEGGER](https://www.segger.com/downloads/jlink) under the section *J-Link Software and Documentation Pack*. Without it when running the apps you would otherwise get error messages when running the apps like `CouldNotFindJprogDLL`.
+
+In order to access Nordic USB devices with correct permissions *udev* rules need to be set up once. For this purpose [nrf-udev](https://github.com/NordicSemiconductor/nrf-udev) repository has been created, follow instructions there.
 
-# Available apps
+## Installation
 
-The following apps have been created for nRF Connect;
+Download binaries from the [nRF Connect for Desktop product page](https://www.nordicsemi.com/eng/Products/Bluetooth-low-energy/nRF-Connect-for-Desktop#Downloads) on Nordic Semiconductor web pages.
```

Comment:
I'd fix the url to the final one: https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Connect-for-desktop

Reply:
Thanks for noticing, fixed in 82c4061cfa2b2bc495c75778b4f57f7bac832124